addons/stock/models/stock_picking.py

```diff
@@ -1074,10 +1074,10 @@ def _onchange_picking_type(self):
                 lambda m: m.picking_type_id != self.picking_type_id
             ).picking_type_id = self.picking_type_id
             (self.move_ids | self.move_ids_without_package).company_id = self.company_id
-            for move in (self.move_ids | self.move_ids_without_package):
-                if not move.product_id:
-                    continue
-                move.description_picking = move.product_id._get_description(move.picking_type_id)
+            # for move in (self.move_ids | self.move_ids_without_package):
```

Comment:
You comment those lines in the first picking and uncomment them in the second picking.
Don't comment them at all in the first picking instead please

Reply:
@mohammadhossam Please don't mark as resolved a comment if you didn't do the change.
PS: I edited my previous comment, I wanted to mean "the first commit", not "the first picking" :grimacing:

Reply:
Oh, I didn't notice that the change was split across both commits. I removed the change completely :+1: